src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet9Isolated

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0-preview-noble-amd64 AS installer-env
+
+COPY . /src/dotnet-function-app
+RUN cd /src/dotnet-function-app && \
+mkdir -p /home/site/wwwroot && \
+dotnet nuget add source https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsTempStaging/nuget/v3/index.json && \
```

Comment:
This feed reference should not make it into the final version.

Reply:
removed